gradle/component.gradle

```diff
@@ -170,7 +171,7 @@ afterEvaluate {
 }
 
 ktlint {
-    disabledRules=["no-wildcard-imports","filename"]
+    disabledRules=["no-wildcard-imports","filename","import-ordering"]
```

Comment:
Check what is going on here exactly and how to fix it without running `ktFormat` so it can hopefully be removed.